sources/dicom/large_image_source_dicom/web_client/templates/assetstoreImport.pug

```diff
@@ -21,6 +21,13 @@ form.g-dwas-import-form
       .input-group-btn
         button.g-open-browser.btn.btn-default(type="button")
           i.icon-folder-open
+    label(for="g-dwas-import-limit") Limit (Studies)
+    input#g-dwas-import-limit.form-control(
+        type="number", step="1", min="1", value="100",
+        onkeypress="return event.charCode >= 48 && event.charCode <= 57")
```

Comment:
The `"number"` type largely does this (but doesn't force it to integers.  Instead of checking the keys, I would raised a validation error on submit by checking if the input, when trimmed, was either empty or a parseable int.  The key check doesn't prevent pasting floating point number into the field, so I'm not sure it helps that much.

Reply:
Actually, it looks like it is already performing some validation somewhere. If I enter in a float or a negative number, it won't let me submit the form and presents errors like the following:

![image](https://github.com/girder/large_image/assets/9558430/fda09598-a688-497a-abe3-b8f1036dff5d)

![image](https://github.com/girder/large_image/assets/9558430/58227c16-8927-4454-b19c-97a9826543b7)

So I'll just remove that `onkeypress` argument and leave it the way it is!

Reply:
That is all coming from the browser via the type="number" and step, etc. values

Reply:
Done!